ngraph/core/reference/include/ngraph/runtime/reference/deformable_convolution.hpp

```diff
@@ -0,0 +1,246 @@
+//*****************************************************************************
+// Copyright 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/runtime/reference/convolution.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            namespace def_conv_impl
+            {
+                inline void validate_params(const Shape& in_shape,
+                                            const Shape& o_shape,
+                                            const Shape& f_shape,
+                                            const Strides& strides,
+                                            const Strides& dilations,
+                                            const CoordinateDiff& pads_begin,
+                                            const CoordinateDiff& pads_end)
+                {
+                    // this implementation supports 2D deformable convolutions
```

Comment:
TODO: Extend `validate_params` implementation to cover checks for dimensions related to groups and deformable values

Reply:
As we agreed in offline discussion extension of checks in validate_params() will be done as a part of operator class review which will be handled in next PR.